cmd/osbuild-composer/config.go

```diff
@@ -15,6 +15,9 @@ type ComposerConfigFile struct {
 		AllowedDomains []string `toml:"allowed_domains"`
 		CA             string   `toml:"ca"`
 	} `toml:"worker"`
+	CloudAPI struct {
```

Comment:
I wonder, I think we never exposed this name, should we perhaps rename this to `ComposerAPI` or something like that (and similarly for the actual code)? "Cloud" sounds... wrong. @ondrejbudai @larskarlitski?

Reply:
Yeah, I really like `ComposerAPI` as well. Thanks!

Reply:
Agreed